docs/src/data/component-status.yaml

```diff
@@ -0,0 +1,610 @@
+- component: Accordion
+  figma: Designing
+  react: Released
+  ios: n/a
+  android: n/a
+  docs: Released
+  group: Structure
```

Comment:
I liked the idea of having components separate from their group, so changing groups would be much easier. Similarly with the roadmap structure -- easier for me to conceptualize each item separately and just changing the quarter for that item, rather than having to move around the entire structure of the data.

For my concept of the content model, it's a field on a given component. 

I think the structure you proposed can make sense with the roadmap, where updates will happen quarterly and involve a lot of mental shifting around.

But with components, I think of the group of the component as something external to the component itself. And if I want to update a component, I find it easier to think alphabetically, rather than having to remember where each component is within the structure we gave it. I can search a file, so that's not so strong, but the structure still seems separate from the component rather than intrinsic to how I think of it.

Reply:
Ok, makes sense 👍